ui/druid/balance/sim.ts

```diff
@@ -106,16 +104,16 @@ const SPEC_CONFIG = registerSpecConfig(Spec.SpecBalanceDruid, {
 	},
 
 	presets: {
-		epWeights: [Presets.P1_EP_PRESET],
+		epWeights: [Presets.StandardEPWeights],
 		// Preset talents that the user can quickly select.
 		talents: [Presets.StandardTalents],
-		rotations: [Presets.PresetRotationDefault],
+		rotations: [Presets.T11PresetRotation, Presets.T12PresetRotation],
 		// Preset gear configurations that the user can quickly select.
-		gear: [Presets.PreraidPresetGear, Presets.T11PresetGear, Presets.T12PresetGear],
+		gear: [Presets.PreraidPresetGear, Presets.T11PresetGear, Presets.T12PresetGear, Presets.T13PresetGear],
 	},
```

Comment:
You could also add a `builds` preset, which will allow for gear + rotation swapping in 1 click. You can check Feral or Fury for examples.

Reply:
also done. i'm not sure why my first preset button doesn't get 'activated' though, might be some cache that's being weird locally..

Reply:
Ah, I was talking about the `builds` array which allows you to swap Gear + Rotation + EP Weights etc with 1 button press